k9mail/src/main/java/com/fsck/k9/activity/setup/outgoing/AccountSetupOutgoing.java

```diff
@@ -437,47 +375,35 @@ private void updateAuthPlainTextFromSecurityType(ConnectionSecurity securityType
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (resultCode == RESULT_OK) {
```

Comment:
One of my questions is, does the logic here need to be put in presenter, too? If so, several methods scattered in presenter and view will be produced, and each methods only contains up to 1~5 lines. It will make the code hard to understand.